swarm/network/stream/stream.go

```diff
@@ -101,22 +87,19 @@ type Registry struct {
 // RegistryOptions holds optional values for NewRegistry constructor.
 type RegistryOptions struct {
 	SkipCheck       bool
-	Syncing         SyncingOption   // Defines syncing behavior
-	Retrieval       RetrievalOption // Defines retrieval behavior
```

Comment:
isn't this somehow necessary for light nodes too?

Reply:
Good question. You see which tests are being removed, and all other tests are passing.

Reply:
It would have been nice to have at least one test that starts a `lightnode` client, a `regular` client and fetches content from the lightnode. I am not aware of such a test, and since Travis is passing, I assume that we are not breaking anything.